app.py

```diff
@@ -231,21 +237,32 @@ def show_column(dstats, ds_name_to_dict, show_embeddings, show_perplexities, col
         The function displays the information using the functions defined in the st_utils class.
     """
     # Note that at this point we assume we can use cache; default value is True.
+    widget_dict = {1: ("general stats", st_utils.expander_general_stats), 2: (
+    "label distribution", st_utils.expander_label_distribution),
+                   3: ("text_lengths", st_utils.expander_text_lengths),
+                   4: ("duplcates", st_utils.expander_text_duplicates),
+                   5: ("npmi", st_utils.npmi_widget),
+                   6: ("zipf", st_utils.expander_zipf)}
+
```

Comment:
It makes it a lot easier to just do a "for" loop with a single line to run each module, rather than cut+paste the same thing over and over. Especially important if there are going to be any further changes.